dts/bindings/mfd/nxp,sc18is606.yaml

```diff
@@ -0,0 +1,33 @@
+description: |
+  NXP SC18IS606 I2C to SPI/GPIO bridge.
+
+  The SC18IS606 is supports both an external SPI and GPIO controller.
+  These controllers have to be added  to the Device Tree as children, While
+  the device itself has to be a child of a I2C controller.
+
+  An example configuration:
```

Comment:
probably worth documenting it's 3 child devices max.

Reply:
Ah yes